src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -88,6 +105,27 @@ private JObject GetInvokeResult(byte[] script, Signers signers = null)
             return json;
         }
 
+        private static JObject ToJson(TreeNode<UInt160> node)
+        {
+            JObject json = new();
+            json["hash"] = node.Item.ToString();
+            if (node.Children.Any())
+            {
+                json["call"] = new JArray(node.Children.Select(ToJson));
+            }
+            return json;
+        }
+
+        private static JObject ToJson(System.Collections.Generic.IEnumerable<DataCache.Trackable> changes)
+        {
+            JArray json = new();
+            foreach (var entry in changes)
+            {
+                json[entry.Key.ToArray().ToHexString()] = Convert.ToBase64String(entry.Item.Value);
```

Comment:
This differs from `StatesDumper` format, do we have any good reason for this difference?

Reply:
None, fixed :)